frontend/src/app/offers/offer-detail/offer-detail.component.ts

```diff
@@ -0,0 +1,29 @@
+import { environment } from '../../../environments/environment';
+import { observable } from 'rxjs/symbol/observable';
```

Comment:
What is this one used for?

Reply:
Being honest, I don't know. Didn't realise it's here. Don't recall adding it, could my IDE done that? I suppose I can safely delete it. We already have Observable import and this one is not used anywhere.